includes/class-newspack-newsletters-renderer.php

```diff
@@ -730,8 +732,8 @@ public static function render_mjml_component( $block, $is_in_column = false, $is
 
 				$block_mjml_markup = '<mj-text ' . self::array_to_attributes( $text_attrs ) . '>' . $inner_content[0];
 				if ( ! empty( $inner_blocks ) && 1 < count( $inner_content ) ) {
-					foreach ( $inner_blocks as $block ) {
-						$block_mjml_markup .= $block['innerHTML'];
+					foreach ( $inner_blocks as $inner_block ) {
+						$block_mjml_markup .= self::render_mjml_component( $inner_block );
```

Comment:
`<mj-text />` components are not supposed to contain other elements. Although rendered it appears fine, the markup is broken with `mj-*` elements:

```html
<ul>
  <mj-section padding="0"
    ><mj-column padding="12px" width="100%"
      ><mj-text
        padding="0"
        line-height="1.5"
        font-size="16px"
        font-family="Georgia"
        ><li>
          List<mj-section padding="0"
            ><mj-column padding="12px" width="100%"
              ><mj-text
                padding="0"
                line-height="1.5"
                font-size="16px"
                font-family="Georgia"
                ><ul>
                  <mj-section padding="0"
                    ><mj-column padding="12px" width="100%"
                      ><mj-text
                        padding="0"
                        line-height="1.5"
                        font-size="16px"
                        font-family="Georgia"
                        ><li>Sub item</li></mj-text
                      ></mj-column
                    ></mj-section
                  >
                </ul></mj-text
              ></mj-column
            ></mj-section
          >
        </li></mj-text
      ></mj-column
    ></mj-section
  >
</ul>
```

Reply:
[5b5b80c](https://github.com/Automattic/newspack-newsletters/pull/1016/commits/5b5b80c55aede86ec55c8f046520bd38997d862f) should avoid nesting MJML elements inside `mj-text`.

Reply:
[b62bd7e](https://github.com/Automattic/newspack-newsletters/pull/1016/commits/b62bd7efd76793d761b9a897367f22237e102244) fixes a regression caused by the previous commit which affected Group blocks and reusable blocks.